Source/CarthageKit/Git.swift

```diff
@@ -167,6 +167,27 @@ public func launchGitTask(
 	let taskDescription = Task("/usr/bin/env", arguments: [ "git" ] + arguments, workingDirectoryPath: repositoryFileURL?.path, environment: updatedEnvironment)
 
 	return taskDescription.launch(standardInput: standardInput)
+		.filter { taskEvent in
+			switch taskEvent {
+			case .success(_):
+				return true
+			case .launch(_):
+				return true
+			case let .standardOutput(data):
+				return data.count > 0
+			case let .standardError(data):
+				return data.count > 0
+			}
+		}
+		.on(event: { event in
+			guard let taskEvent = event.value else {
+				return
+			}
+			if case .success(_) = taskEvent {
+				return
+			}
+			print(taskEvent)
+		})
 		.ignoreTaskData()
```

Comment:
@mdiep OK, so few issues.

1. launchGitTask returns a `SignalProducer<String, CarthageError>`. So all of the places that expect a string of the success output would have to change.
2. And even if we did that, we still wouldn't be able to let *all* of the git output be logged. Because there are some instances like in `listTags` where they need to operate on the successful string data and map into something else.

What would you think about adding another optional parameter to the launchGitTask func that is an observer/event sink. Then if verbose, we push to stdout, else we ignore.